website/src/docs/bananacakepop/index.md

```diff
@@ -2,8 +2,18 @@
 title: "Introduction"
 ---
 
-You are looking for the download version of our GraphQL IDE?
+Banana Cake Pop is our brand new GraphQL IDE which works well with Hot Chocolate and any other GraphQL server. It's still in preview so try it out and give us feedback via our slack channel.
 
-We are getting ready for the next big release of Banana Cake Pop. You can already try out the preview version [here](http://workshop.chillicream.cloud).
+![Banana Cake Pop - Operations](../shared/bcp/bcp-operations.png)
 
-The new download version should be available in the next couple of days.
+![Banana Cake Pop - Schema Reference](../shared/bcp/bcp-schema-reference.png)
+
+![Banana Cake Pop - Schema Definition](../shared/bcp/bcp-schema-definition.png)
+
+# Download
+
+Banana Cake Pop is currently in preview and the current version is `1.0.0-preview.1`.
```

Comment:
Make a note of auto-update, so people know they don't have to check some site for an update.